src/test/java/random_csapatnev/BearInfectsVirologistStepDefs.java

```diff
@@ -0,0 +1,80 @@
+package random_csapatnev;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import random_csapatnev.modelclasses.*;
+import random_csapatnev.modelclasses.Character;
+import random_csapatnev.viewclasses.GraphicsCharacter;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+import java.util.ArrayList;
+
+class CouldHeInfectVirologist {
+    static String couldHeInfectVirologist(boolean success) {
+        return success ? "Yes I could" : "No I couldn't";
+    }
+}
+
+public class BearInfectsVirologistStepDefs {
+    private Virologist virologist = new Virologist("Virologist");
+    private Bear bear = new Bear("bBear");
```

Comment:
Typo? ("bBear")

Reply:
Az nem typo. Ilyen módon van kezelve a bearInteract() és nem akartam hogy saját magával is interactoljon.
![image](https://github.com/BME-MIT-IET/iet-hf-2023-grafadatbazis/assets/43915301/51739714-e67f-47ee-b16c-e707dde61c67)